explorer/src/components/account/UnknownAccountCard.tsx

```diff
@@ -34,7 +34,11 @@ export function UnknownAccountCard({ account }: { account: Account }) {
         <tr>
           <td>Balance (SOL)</td>
           <td className="text-lg-end">
-            <SolBalance lamports={account.lamports} />
+            {account.lamports === 0 ? (
+              "No longer exists on chain"
```

Comment:
This message is inaccurate and misleading. The account may have never existed on chain in the first place. Can this be clarified in a follow-up PR please.

Reply:
What would you like the message to be?

e.g. "The account no longer exists on chain, or may have never existed on chain"?

Reply:
"Account does not exist" perhaps?